### DISCUSSION TITLE
Wind Farm case

### DISCUSSION THREAD

**Discussion by Shail546** [turbinesFoam/turbinesFoam#337] [General]
Hello Sir, 
Is it possible to extend this case to a a Wind Farm containing many turbines?
Please guide me regarding it.
Thank you.

**Comment by petebachant** [comment]
Sure, it's certainly possible. In its current form, you'd need to create the farm as a bunch of turbines with different names in `fvOptions`, since there is no higher level farm object, though that would be nice to add someday.
